modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/web/internal/info/item/provider/test/ObjectEntryInfoItemFieldValuesProviderTest.java

```diff
@@ -131,6 +127,8 @@ public void setUp() throws Exception {
 				Collections.singletonList(
 					ListTypeEntryUtil.createListTypeEntry(_listTypeEntryKey)));
 
+		_mockHttpServletRequest = new MockHttpServletRequest();
```

Comment:
@vngoliveira this variable is only used by one test `testObjectEntryInfoItemFieldValuesProvider`.

That said, I see no reason to make it global and add it here in the `setUp` method.